web/themes/custom/my_barrio/templates/form-element.html.twig

```diff
@@ -0,0 +1,160 @@
+{#
```

Comment:
Test the site to check if any other form is updated by this template.

Reply:
Yes, all forms on site are also updated. I removed the file and made changes in css.